### ISSUE TITLE
Permit Rectangular Brushes 1 Square Wide/High

### ISSUE THREAD

**Issue by Alphanos** [TEdit/Terraria-Map-Editor#217]
I know that this wouldn't make sense for ellipses or outline mode.  However the ability to paint 1 square wide/high with rectangular brushes would be very useful.

For example, right now if I wanted to draw a floor/wall 50 squares wide and only 1 square high, it takes a lot of steps.  I could use the pencil, which would also require a lot of undos when I don't move my hand perfectly straight.  Or I could draw just a 10 square section with the pencil and copy/paste it 4 more times.  

Basically there are many scenarios where a 1 square wide/high rectangular brush would be very helpful :-).
